src/IconTools/index.js

```diff
@@ -12,7 +12,8 @@ import {
   faDrawPolygon,
   faVectorSquare,
   faHandPaper,
-  faSearch
+  faSearch,
+  faCircle
```

Comment:
we should find a dotted circle outline if one is available, the solid circle isn't quite clear IMO

Reply:
I agree. @fortawesome/free-solid-svg-icons has solid icons. Shall I add @fortawesome/free-regular-svg-icons to the package.json and use the non solid circle?

Reply:
let's do it :)